onnxruntime/python/tools/transformers/onnx_model_bert.py

```diff
@@ -48,7 +48,6 @@ def __init__(self, model: ModelProto, num_heads: int = 0, hidden_size: int = 0):
             num_heads (int, optional): number of attention heads. Defaults to 0 (detect the parameter automatically).
             hidden_size (int, optional): hidden dimension. Defaults to 0 (detect the parameter automatically).
         """
-        assert (num_heads == 0 and hidden_size == 0) or (num_heads > 0 and hidden_size % num_heads == 0)
```

Comment:
it's better to override the init() in onnx_model_t5.py. onnx_model_bert.py has the base class used in other files